package.json

```diff
@@ -27,6 +27,9 @@
 		"types": "tsc --project ./tsconfig.build.json",
 		"validate": "npm-run-all --parallel tsc lint 'test --onlyChanged'"
 	},
+	"dependencies": {
+		"@guardian/old-cmp": "npm:@guardian/consent-management-platform@^3.4.10"
```

Comment:
Be aware 3.4.10 is compatible with Source v2.0!

Reply:
this is the version currently shipping in `4.0.*` – we should probably match that?

Reply:
Great!